packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,213 @@
+- `tertiaryText`: Third line of text in Persona.
+- `quaternaryText`: Fourth line of text in Persona.
+
+**Types**
+
+```ts
+export type PersonaSlots = {
+  root: NonNullable<Slot<'div'>>;
+
+  /**
+   * Avatar to display.
+   */
+  avatar?: Slot<typeof Avatar>;
+
+  /**
+   * PresenceBadge to display.
+   */
+  presence?: Slot<typeof PresenceBadge>;
+
+  /**
+   * The first line of text in the Persona, larger than the rest of the lines.
+   *
+   * This defaults to the `name` prop, and it is recomended that you only set its value if it should be different from
+   * from the `name` prop.
+   */
+  primaryText?: Slot<'span'>;
+
+  /**
+   * The second line of text in the Persona.
+   */
+  secondaryText?: Slot<'span'>;
+
+  /**
+   * The third line of text in the Persona.
+   */
+  tertiaryText?: Slot<'span'>;
+
+  /**
+   * The fourth line of text in the Persona.
+   */
+  quaternaryText?: Slot<'span'>;
+};
+
+/**
+ * Persona Props
+ */
+export type PersonaProps = Omit<ComponentProps<PersonaSlots>, 'badge'> &
+  Pick<AvatarProps, 'name'> & {
+    /**
+     * Whether to display only the presence.
+     *
+     * @default false
+     */
+    presenceOnly?: boolean;
+
+    /**
+     * Position the text will be rendered in.
+     *
+     * @default after
+     */
+    textPosition?: 'before' | 'after' | 'below';
+  };
+```
 
 ## Structure
 
-- _**Public**_
+To avoid the [issue](https://github.com/microsoft/fluentui/issues/23386) v8 has, a css grid will be used instead of a flexbox that requires a general wrapper and a text container wrapper.
+
 - _**Internal**_
-- _**DOM** - how the component will be rendered as HTML elements_
 
-## Migration
+```jsx
+const coin = (
+  <>
+    {!presenceOnly && slots.avatar && <slots.avatar {...slotProps.avatar} />}
+    {presenceOnly && slots.presence && <slots.presence {...slotProps.presence} />}
+  </>
+);
+
+return (
+  <slots.root {...slotProps.root}>
+    {(textPosition === 'after' || textPosition === 'below') && coin}
+    {slots.primaryText && <slots.primaryText {...slotProps.primaryText} />}
+    {slots.secondaryText && <slots.secondaryText {...slotProps.secondaryText} />}
+    {slots.tertiaryText && <slots.tertiaryText {...slotProps.tertiaryText} />}
+    {slots.quaternaryText && <slots.quaternaryText {...slotProps.quaternaryText} />}
+    {textPosition === 'before' && coin}
+  </slots.root>
+);
+```
+
+- _**DOM**_
+
+```html
+<div class="fui-Persona">
+  <div {/* Avatar, PresenceBadge, or Avatar with PresenceBadge */} />
+  <span class="fui-Persona__primaryText">{/* name */}</span>
+  <span class="fui-Persona__secondaryText">Secondary Text</span>
+  <span class="fui-Persona__tertiaryText">Tertiary Text</span>
+  <span class="fui-Persona__quaternaryText">Quaternary Text</span>
+</div>
+```
 
-_Describe what will need to be done to upgrade from the existing implementations:_
+## Migration
 
-- _Migration from v8_
-- _Migration from v0_
+See [MIGRATION.md](./MIGRATION.md) for details.
```

Comment:
It doesn't look like MIGRATION.md exists yet. Do you plan to add it in another PR?

Reply:
yes, I was planning on adding it with this issue: #24956